test/functional/bootstrap-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+// transpile :mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import path from 'path';
+import AndroidBootstrap from '../../index.js';
```

Comment:
indeed. also, our convention with our es6 code is not to add `.js`. so while we should go with isaac's suggestion here, if we weren't, this should look like `'../../index'`